shared_models/shared_models.py

```diff
@@ -239,6 +241,7 @@ class PresentationExchange(BaseModel):
     ] = None
     updated_at: Optional[str] = None
     verified: Optional[bool] = None
+    thread_id: Optional[str] = None
```

Comment:
I thiught thread_id is not optional for most models, can you check?

Reply:
yes will check. However, I made that optional to cover the case where this is None for strange/unknown reasons we've seen before in the webhooks.

Reply:
at least in the cloudcontroller api this is optional

Reply:
Yeah true, but having an always defined property is a lot nicer to work with than an optional attribute, so if we can prevent it being optional that would have my preference

Reply:
> at least in the cloudcontroller api this is optional

Ah okay, maybe keep it optional then. I think the cloud controller says a lot of properties are optional that in fact, aren't.

Reply:
yeah indeed. maybe that's something to get to the bottom of (in the cloudcontroller) as well.

Reply:
I _think_ it's because ACA-Py doesn't mark them as required, so we would have to fix this in ACA-Py itself 🤷

Reply:
interesting btw - if they're optional in aca-py itself maybe there is a reason these are Optional? Or it's just being too permissive in the models...